test/spec_helper.rb

```diff
@@ -52,4 +52,10 @@
 
 RSpec.configure do |c|
   c.include Yast::RSpec::StorageHelpers
+
+  c.before do
+    # Storage::Storage#activate should not be called during testing because we are not
+    # doing real probing.
+    allow_any_instance_of(Storage::Storage).to receive(:activate).and_return(true)
```

Comment:
An acceptable workaround hopefully to be removed when completing https://trello.com/c/BWtX9ccg/548-2-remove-refinements-patches-extensions-etc-from-yast-storage-ng